### DISCUSSION TITLE
Help: Minecraft/Fabric: "How to create blocks(furniture) that rotate 45*,135*,225*,315* "

### DISCUSSION THREAD

**Discussion by UnintelligentIdiot** [bernie-g/geckolib#485] [Q&A]
The idea is that the block(furniture) would rotate like the heads(Steve head ,skeleton head etc.) as well as able to make them have an opening animation for chests. How can I do that?

I imagine having the block rotate like the head on the ground(meaning the title and 90*,180*,270) as well as having a separate model for when placing it on a wall as well as on ceiling(same thing applies here as well)

I tried with putting the furniture in invisible item frame and a barrier block(like a data pack) but it lags a lot when I have more stuff and when I get a little far away the entity disappears

**Comment by Tslat** [comment]
You make a blockstate that has rotations like that, then you manipulate it in the renderer

**Reply by UnintelligentIdiot** [reply]
How do I do that cuz I was trying so hard with tutorials and everything but couldn't figure it out

**Reply by Tslat** [reply]
Ask in a general modding server
Geckolib is not a general modding help platform

**Reply by UnintelligentIdiot** [reply]
ohh sorry didn't know
